esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -4320,10 +4320,17 @@ void CWsWorkunitsEx::createZAPWUInfoFile(IEspWUCreateZAPInfoRequest &req, Owned<
     StringBuffer sb;
     sb.append("Workunit:     ").append(cwu->queryWuid()).append("\r\n");
     sb.append("User:         ").append(cwu->queryUser()).append("\r\n");
-    sb.append("Build Version:").append(req.getBuildVersion()).append("\r\n");
+    sb.append("Build Version:").append(getBuildVersion()).append("\r\n");
     sb.append("Cluster:      ").append(cwu->queryClusterName()).append("\r\n");
     if (req.getESPIPAddress())
         sb.append("ESP:          ").append(req.getESPIPAddress()).append("\r\n");
```

Comment:
Is there a reason to continue trying to query it from the request, instead of always executing your new code below?